### PR TITLE
#1336 add Invoice Item id as param valid in account item-detail command

### PR DESCRIPTION
**Pull request by ATGE**

### CODE REVIEW

File: SoftLayer/managers/account.py [hunk 1]
```diff
@@ -219,6 +220,28 @@ def get_billing_item(self, identifier, mask=None):
 
         return self.client.call('Billing_Item', 'getObject', id=identifier, mask=mask)
 
+    def get_billing_item_from_invoice(self, identifier):
+        """Gets details about a billing item of a billing invoice item
+
+        :param int identifier: Billing_Invoice_Item id
+        :return: Billing_Item
+        """
+        return self.client.call('Billing_Invoice_Item', 'getBillingItem', id=identifier)
```
Comment (allmightyspiff) [inline SoftLayer/managers/account.py:229]: Should be using the same mask as `get_billing_item`, specifically `,order[id,userRecord[id,email,displayName,userStatus]]]` is missing.

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, but `get_billing_item_from_invoice` needs to pull in the order->user information. Otherwise everything will show up as ordered by IBM.

**Comment by allmightyspiff** [comment]
## TESTING

```
$ ./slcli --format=table  -v account item-detail 1677911496
Calling: SoftLayer_Billing_Item::getObject(id=1677911496, mask='mask[ orderItem[id,order[id,userRecord[id,email,displayName,userStatus]]], nextInvoiceTotalRecurringAmount, location, hourlyFlag, children ]', filter='None', args=(), limit=None, offset=None))
Calling: SoftLayer_Billing_Invoice_Item::getBillingItem(id=1677911496, mask='', filter='None', args=(), limit=None, offset=None))
:...............................................:
:           1 x 2.0 GHz or higher Core          :
:..................:............................:
:       Key        :           Value            :
:..................:............................:
:    createDate    :         2020-04-07         :
:  cycleStartDate  :         2020-09-03         :
: cancellationDate :            None            :
:   description    : 1 x 2.0 GHz or higher Core :
:       FQDN       :    activedirectory.None    :
:   recurringFee   :             0              :
:    Ordered By    :            IBM             :
:      Notes       :            None            :
:     Location     :            None            :
:..................:............................:
```

File: SoftLayer/managers/account.py (hunk 1 above)
Comment (ATGE) [reply SoftLayer/managers/account.py]: Updated

**Review by allmightyspiff — APPROVED** [review]

**Comment by allmightyspiff** [comment]
Changes look good, thanks.
Not sure why analysis started failing, but i'll fix that in #1347 and then merge this.

**Pull request merged**